installer/bundle/bundle_skel.sh

```diff
@@ -553,16 +552,6 @@ do
             shift 1
             ;;
 
-        --collectd)
```

Comment:
Maybe we could keep this condition, but if this parameter is used, just print an informational message like "CollectD can no longer be configured with the --collectd option. Please see OMSAgent-for-Linux documentation here: https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/OMS-Agent-for-Linux.md"